src/packages/commit-lint/lib/lib.js

```diff
@@ -0,0 +1,70 @@
+const fs = require('fs');
+const { ArgumentParser } = require('@eigenspace/argument-parser');
+
+function exitWithSuccess(message) {
+    // eslint-disable-next-line no-console
+    console.info(message);
+    process.exit(0);
+}
+
+function exitWithError(message) {
+    // eslint-disable-next-line no-console
+    console.error(message);
+    process.exit(1);
+}
+
+function getCommitMessage() {
+    const parser = new ArgumentParser();
+    const args = parser.get(process.argv);
+
+    const message = args.get('message');
+
+    if (message === 'COMMIT_MESSAGE') {
+        return fs.readFileSync('.git/COMMIT_EDITMSG', 'utf8');
+    }
+
+    return message;
+}
+
+function getConfigContentFrom(path) {
+    let config = {};
+    try {
+        const content = fs.readFileSync(path, 'utf8');
+        config = JSON.parse(content);
+    } catch (err) {
+        exitWithError(`Failed to load ${path}`);
+        return;
+    }
+
+    return config;
+}
+
+function merge(...objects) {
```

Comment:
It is either candidate to be moved to `@eigenspace/utils` or to be replaced with `@lodash/merge` (but array merge works different there).

Reply:
Planned to the next release